config/defaults.php

```diff
@@ -24,6 +24,9 @@
             ]
         ],
         'timezoneAwareDateTimeWidget' => false,
-        'useAssetCompress' => Plugin::loaded('AssetCompress')
+        'useAssetCompress' => Plugin::loaded('AssetCompress'),
+        'tables_blacklist' => [
```

Comment:
All `scaffold.*` action config are underscored but all `Crud.*` global configs are variable cased. I'll change it.